include/swift/ClangImporter/CXXMethodBridging.h

```diff
@@ -62,16 +62,27 @@ struct CXXMethodBridging {
   }
 
   NameKind classifyNameKind() {
-    bool allLower = llvm::all_of(getClangName(), islower);
+#if defined(__linux__)
```

Comment:
What's the issue here? Linux/CentOS?

Reply:
Oh, I bet this is an issue with `isupper`. Is that correct? You probably need to include `cctype`.